docker/kites/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:latest
+MAINTAINER Simone Zaccariello <simone.zaccariello@gmail.com>
+# install binary and remove cache
+RUN apt-get update \
+    && apt-get install -y iperf3 netsniff-ng nano tcpdump screen \
+    && rm -rf /var/lib/apt/lists/*
+
+# Expose the default iperf3 server port
+EXPOSE 5201
+
+# entrypoint allows you to pass your arguments to the container at runtime
+# very similar to a binary you would run. For example, in the following
+# docker run -it <IMAGE> --help' is like running 'iperf3 --help'
+ENTRYPOINT ["iperf3"]
+CMD ["-s"]
```

Comment:
I would avoid adding this CMD. At this point, we want to keep this image as generic as possible, so that can be used to deploy a client or a server.

Reply:
Hi Marco, CMD can be overriden so leaving -c as the default seems ok, take also into account that running it as a server is required to have the container persist after the run, so it will be used as a server in our use case

Reply:
Yes, but we can specify the behaviour in the Pod descriptor. No need to set this here.

Reply:
I would prefer to leave the container with the default -c option, so if one runs it manually he does not have to remember to put the -c option 
the -c CMD option can be overridden both when running manually and by specifying the behavior in the pod descriptor
(note that this is a theoretical discussion, as we simply need this container in server mode so that it is persistent)

Reply:
ok, @StefanoSalsano you won :)